src/packages/Highlighter/Node/HighlighterNode.ts

```diff
@@ -1,5 +1,5 @@
 import randomcolor from 'randomcolor';
-import { HighlighterCore } from '../core';
+import { HighlighterCore } from '../Core';
```

Comment:
Filename and classname do not match folder name
`/Node/HighlighterNode.ts` --> `/HighlighterNode/HighlighterNode.ts` 

Same for 
`Core/HighlighterCore.ts` --> `HighlighterCore/HighlighterCore.ts`

Reply:
Done!